packages/js/product-editor/src/blocks/inventory-sku/block.json

```diff
@@ -0,0 +1,26 @@
+{
+	"$schema": "https://schemas.wp.org/trunk/block.json",
+	"apiVersion": 2,
+	"name": "woocommerce/product-sku",
+	"title": "Product text control",
+	"category": "woocommerce",
+	"description": "The product sku.",
+	"keywords": [ "products", "sku" ],
+	"textdomain": "default",
+	"attributes": {
+		"name": {
```

Comment:
Just following @joshuatf's suggestions, see -> https://github.com/woocommerce/woocommerce/pull/37683#discussion_r1164497151

Reply:
Thanks @mdperez86 , I just [responded in that thread](https://github.com/woocommerce/woocommerce/pull/37683#discussion_r1166040083).